chirp/drivers/kenwood_live.py

```diff
@@ -121,13 +121,18 @@ def get_id(ser):
             ser.baudrate = i
             ser.write(LAST_DELIMITER[0].encode())
             ser.read(25)
-            resp = command(ser, "ID")
+            try:
+                resp = command(ser, "ID")
+            except UnicodeDecodeError:
+                LOG.info("Unknown utf-8 byte received at baud %i with \
+                        delimiter \"%s\"" % (i, repr(delimiter)))
+                LOCK.release()
```

Comment:
This is a bad idea, because it makes it easier for the lock to get out of sync. We should use a try..finally in command so that it is always dropped before we exit, even if we raise. This is a latent bug in the current implementation, which could be tickled if we hit other exceptions, so it's good that you highlighted it here.